security/security-spring/src/main/java/org/cbioportal/security/spring/CancerStudyPermissionEvaluator.java

```diff
@@ -181,6 +182,15 @@ public boolean hasPermission(Authentication authentication, Serializable targetI
                 }
             }
             return true;
+        } else if ("List<SampleListId>".equals(targetType)) {
```

Comment:
The implementation of this loop and other permission checking of various lists results in behavior such that if a single ID in a list is unauthorized, then nothing is returned.  Will this ever happen and if so, is this the behavior we want?  The alternative is to silently drop unauthorized ids.  I just wanted to raise the issue for discussion-consensus.